test/github_changelog/github_test.clj

```diff
@@ -7,6 +7,10 @@
              [schema-generators :as sgen]]
             [jsonista.core :as j]))
 
+(deftest http-get
+  (is (= {:href "https://api.github.com/gists?page=2"}
```

Comment:
This will be a super flaky tests, see the Travis output.

Reply:
Perhaps we can use an api key and set up Travis to provide it?

Reply:
I switched to a third-party site that does not have rate limiting to mock the GitHub API response.